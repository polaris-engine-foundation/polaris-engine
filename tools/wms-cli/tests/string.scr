func main() {
     a = "str1";
     b = "str2";
     c = a + b + "str3";
     print(a);
     print(b);
     print(c);

     a = a + 1;
     print(a);

     b = b + "abc";
     print(b);

     c = "" + 1;
     print(c);
}
