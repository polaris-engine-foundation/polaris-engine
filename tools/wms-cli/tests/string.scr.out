str1
str2
str1str2str3
str11
str2abc
1
